go.mod

```diff
@@ -3,13 +3,21 @@ module github.com/kubernetes-csi/livenessprobe
 go 1.16
 
 require (
-	github.com/container-storage-interface/spec v1.5.0
+	github.com/container-storage-interface/spec v1.6.0
+	github.com/go-logr/logr v1.2.3 // indirect
 	github.com/golang/mock v1.5.0
 	github.com/kubernetes-csi/csi-lib-utils v0.11.0
 	github.com/kubernetes-csi/csi-test/v4 v4.0.0-20200806214950-555d70a11a8b
-	github.com/prometheus/client_golang v1.11.1 // indirect
-	google.golang.org/grpc v1.40.0
-	k8s.io/klog/v2 v2.60.0
+	github.com/prometheus/client_golang v1.12.2 // indirect
+	github.com/prometheus/common v0.37.0 // indirect
+	github.com/prometheus/procfs v0.8.0 // indirect
+	golang.org/x/net v0.0.0-20220805013720-a33c5aa5df48 // indirect
+	golang.org/x/sys v0.0.0-20220804214406-8e32c043e418 // indirect
+	google.golang.org/genproto v0.0.0-20220804142021-4e6b2dfa6612 // indirect
+	google.golang.org/grpc v1.48.0
+	google.golang.org/protobuf v1.28.1 // indirect
+	k8s.io/component-base v0.24.3 // indirect
+	k8s.io/klog/v2 v2.70.1
```

Comment:
So this is mostly just a dependency update, i.e. business as usual. This part looks fine.

But why does go.mod have replace statements for golang.org/x/text? Why was that added and is that still relevant?

Reply:
Actually its not needed, I have reran it again which have removed that mention, ptal.. thanks @pohly

Reply:
That line was intact :) , anyway I have removed that as well @pohly ,